core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -289,6 +291,35 @@ object Uri {
       }
   }
 
+  object Authority {
+    def apply(
+        userInfo: Option[UserInfo] = None,
+        host: Host = RegName("localhost"),
+        port: Option[Int] = None): Authority =
+      new Authority(userInfo, host, port)
```

Comment:
@rossabaker you can drop the `new` here now that we aren't using `AbstractFunction3`.

Reply:
Wait, wouldn't that make a circular definition?

Reply:
@rossabaker yes, sorry. You should be able to drop the whole function, if I'm not mistaken.

Reply:
Oh.  Yes, I think that's true.  Let's see if I still think so after I make the change. :laughing: